pkg/daemon/gpsd.go

```diff
@@ -161,7 +171,16 @@ func (g *GPSD) CmdRun(stdoutToSocket bool) {
 	}
 	g.subscriber = &GPSDSubscriber{source: event.MONITORING, gpsd: g, id: string(event.GNSS)}
 	g.registerSubscriber()
-	processStatus(nil, g.name, g.messageTag, PtpProcessUp)
+	g.stdoutToSocket = stdoutToSocket
+	if !g.stdoutToSocket {
+		processStatus(nil, g.name, g.messageTag, PtpProcessUp)
+	} else {
+		c, err := net.Dial("unix", eventSocket)
```

Comment:
you shouldn;t have to redial; the main daemon handles that . You should only pass it